src/aequitas/bias.py

```diff
@@ -185,5 +183,20 @@ def get_disparity_predefined_groups(self, df, ref_groups_dict, key_columns=None,
         # when there is a zero in the numerator and a zero in denominator it is considered NaN
         # after division, so if 0/0 we assume 1.0 disparity (they are the same...)
         fill_zeros = {metric: 1.000000 for metric in disparity_metrics}
-        #df = df.fillna(value=fill_zeros)
+        # df = df.fillna(value=fill_zeros)
         return df
+
+
+    def list_disparities(self, df):
+        '''
+        View all calculated disparities in table
+        :return: list of disparity metrics
+        '''
+        return list(df.columns[df.columns.str.contains('_disparity')])
+
+    def list_absolute_metrics(self, df):
+        '''
+        View all calculated disparities in table
+        :return: list of absolute group metrics
+        '''
+        return list(set(self.input_group_metrics) & set(df.columns))
```

Comment:
just FYI, I don't see where these new methods are used or needed?

Reply:
I figured that might be the case -- as a rule I don't bother with the notebooks :smile: